solr/core/src/java/org/apache/solr/security/JWTVerificationkeyResolver.java

```diff
@@ -140,16 +155,23 @@ public Key resolveKey(JsonWebSignature jws, List<JsonWebStructure> nestingContex
       }
     } catch (JoseException | IOException | InvalidJwtException | MalformedClaimException e) {
       StringBuilder sb = new StringBuilder();
-      sb.append("Unable to find a suitable verification key for JWS w/ header ").append(jws.getHeaders().getFullHeaderAsJsonString());
-      sb.append(" due to an unexpected exception (").append(e).append(") while obtaining or using keys from source ");
+      sb.append("Unable to find a suitable verification key for JWS w/ header ")
+          .append(jws.getHeaders().getFullHeaderAsJsonString());
+      sb.append(" due to an unexpected exception (")
+          .append(e)
+          .append(") while obtaining or using keys from source ");
       sb.append(keysSource);
       throw new UnresolvableKeyException(sb.toString(), e);
     }
 
     if (theChosenOne == null) {
       StringBuilder sb = new StringBuilder();
```

Comment:
This is a needless use of StringBuilder; but I can understand if you choose to not address it in this PR

Reply:
Used String.format() instead